client-src/elements/chromedash-report-review-latency-page.ts

```diff
@@ -86,13 +86,15 @@ export class ChromedashReportReviewLatencyPage extends LitElement {
   }
 
   renderFeatureReviewsRow(reviewLatency) {
+    const groupedByGateType = Object.groupBy(reviewLatency.gate_reviews, ({gate_type}) => gate_type);
```

Comment:
gate_type is an int and the docs for Object.groupBy() say to only use it with strings.  I have a feeling that it might work in this case, but Map.groupBy() might be better.